src/popup/router/components/SearchBar.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <InputField
+    class="search-bar"
+    :v-model="value"
+    :value="internalValue"
+    v-bind="$attrs"
+    @input="input"
+    type="text"
+  >
+    <Search slot="right" v-if="!internalValue.length" />
+    <Eraser slot="right" v-else @click="input('')" />
```

Comment:
```suggestion
    :value="value"
    v-bind="$attrs"
    @input="$emit('input', $event)"
  >
    <Search slot="right" v-if="!value" />
    <Eraser slot="right" v-else @click="$emit('input', '')" />
```
In general, it looks like that this component can be stateless.